index_maga.html

```diff
@@ -13,26 +13,29 @@
   <link href="specialties.css" rel="stylesheet">
   <link href="delicious__menu__block.css" rel="stylesheet">
   <link href="delicious__menu.css" rel="stylesheet">
+  <link href="private-events.css" rel="stylesheet">
+  <link href="contact-block.css" rel="stylesheet">
+  <link href="footer.css" rel="stylesheet">
 </head>
 <body>
 <div class = "main">
   <div class = "start__screen">
```

Comment:
можно заменить на блок

```suggestion
  <div class = "start">
```

Reply:
"старт" менее понятно, а "начальный экран" можно было назвать первым слайдом, глобальные изменения будут в блоке меню, там можно половину убрать (или почти половину)